ui/page/transaction/transactions_page.go

```diff
@@ -209,99 +276,159 @@ func (pg *TransactionsPage) loadTransactions(offset, pageSize int32) ([]*sharedW
-												return components.LayoutTransactionRow(gtx, pg.Load, wal, tx, true)
-											}),
-											layout.Rigid(func(gtx C) D {
-												// No divider for last row
-												if index == len(wallTxs)-1 {
-													return layout.Dimensions{}
-												}
+			return pg.scroll.List().Layout(gtx, 1, func(gtx C, i int) D {
+				return layout.Inset{Right: values.MarginPadding2}.Layout(gtx, func(gtx C) D {
+					return card.Layout(gtx, func(gtx C) D {
+						if itemCount == -1 || pg.showLoader {
+							gtx.Constraints.Min.X = gtx.Constraints.Max.X
+							return layout.Center.Layout(gtx, pg.materialLoader.Layout)
+						}
 
-												gtx.Constraints.Min.X = gtx.Constraints.Max.X
-												separator := pg.Theme.Separator()
-												return layout.E.Layout(gtx, func(gtx C) D {
-													// Show bottom divider for all rows except last
-													return layout.Inset{Left: values.MarginPadding56}.Layout(gtx, separator.Layout)
-												})
-											}),
-										)
+						wallTxs := pg.scroll.FetchedData()
+						return pg.transactionList.Layout(gtx, len(wallTxs), func(gtx C, index int) D {
+							tx, wal := pg.txAndWallet(wallTxs[index])
+							return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+								layout.Rigid(func(gtx C) D {
+									isHiddenAssetsInfo := true
+									if pg.selectedWallet == nil {
+										isHiddenAssetsInfo = !isHiddenAssetsInfo
+									}
+									return components.LayoutTransactionRow(gtx, pg.Load, wal, tx, isHiddenAssetsInfo)
+								}),
+								layout.Rigid(func(gtx C) D {
+									// No divider for last row
+									if index == len(wallTxs)-1 {
+										return layout.Dimensions{}
+									}
+
+									gtx.Constraints.Min.X = gtx.Constraints.Max.X
+									separator := pg.Theme.Separator()
+									return layout.E.Layout(gtx, func(gtx C) D {
+										// Show bottom divider for all rows except last
+										return layout.Inset{Left: values.MarginPadding56}.Layout(gtx, separator.Layout)
 									})
-								})
-							})
+								}),
+							)
 						})
 					})
-				}),
-				layout.Expanded(func(gtx C) D {
-					return pg.txTypeDropDown.Layout(gtx, 0, true)
-				}),
-			)
+				})
+			})
 		})
-	})
+	})}
+
+	// disabled overlay
+	if pg.walletNotReady() && pg.isHomepageLayout {
+		layoutContents = append(layoutContents, layout.Stacked(func(gtx C) D {
+			gtx = gtx.Disabled()
+			overlayColor := pg.Theme.Color.Gray3
+			overlayColor.A = 220
+			gtx.Constraints.Min.X = gtx.Constraints.Max.X
+			gtx.Constraints.Min.Y = gtx.Constraints.Max.Y - gtx.Dp(values.MarginPadding60)
+			cryptomaterial.FillMax(gtx, overlayColor, 10)
+
+			lbl := pg.Theme.Label(values.TextSize20, values.String(values.StrFunctionUnavailable))
+			lbl.Font.Weight = font.SemiBold
+			lbl.Color = pg.Theme.Color.PageNavText
+			return cryptomaterial.CentralizeWidget(gtx, lbl.Layout)
+		}))
+	}
+
+	return layout.Stack{Alignment: layout.NW}.Layout(gtx, layoutContents...)
```

Comment:
What's the purpose of the NW alignment in this stack?

Also, `layoutContents` may have only one child (if there's no disabled overlay)`. Is it necessary to wrap that single child in a stack layout? I really don't think so. This should work:
```
txListWidget := func(gtx C) D {
	return layout.Inset{Top: values.MarginPadding60}.Layout(gtx, func(gtx C) D {
		...
	}
}

showOverlay := pg.walletNotReady() && pg.isHomepageLayout
if !showOverlay {
	return txListWidget(gtx)
}

return layout.Stack{Alignment: layout.NW}.Layout(gtx, 
	layout.Expanded(txListWidget),
	layout.Stacked(func(gtx C) D {
		gtx = gtx.Disabled()
		overlayColor := pg.Theme.Color.Gray3
		overlayColor.A = 220
		gtx.Constraints.Min.X = gtx.Constraints.Max.X
		gtx.Constraints.Min.Y = gtx.Constraints.Max.Y - gtx.Dp(values.MarginPadding60)
		cryptomaterial.FillMax(gtx, overlayColor, 10)

		lbl := pg.Theme.Label(values.TextSize20, values.String(values.StrFunctionUnavailable))
		lbl.Font.Weight = font.SemiBold
		lbl.Color = pg.Theme.Color.PageNavText
		return cryptomaterial.CentralizeWidget(gtx, lbl.Layout)
	})
)
```

Reply:
What's the purpose of the NW alignment in this stack?: Its used to align the stacked widget.